### PR TITLE
move from CODE to INFRA

### PR DESCRIPTION
**Pull request by JuliaBrigitte**
Since there is no need to have CODE and PROD because this is a low risk infrastructure application to which we can directly deploy, we moved from CODE to INFRA.

We also autogenerate riff-raff.yaml from now on.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -39,11 +39,17 @@ jobs:
         run: ./scripts/ci.sh
 
       - name: Upload to riff-raff
-        uses: guardian/actions-riff-raff@v1.1.3
+        uses: guardian/actions-riff-raff@v2
         with:
```
Comment (kenoir) [inline .github/workflows/ci.yml:43]: I think the indentation here needs fixing.

Comment (kenoir) [reply .github/workflows/ci.yml:43]: Ignore this, reviewing on my phone made this look weird.

**Review by kenoir — APPROVED** [review]

File: packages/cdk/bin/cdk.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-const app = new App();
+const app = new GuRootExperimental();
 
-new GithubLens(app, 'GithubLens-CODE', { stack: 'deploy', stage: 'CODE' });
+new GithubLens(app, 'GithubLens-INFRA', {
+	stack: 'deploy',
+	stage: 'INFRA',
+	env: { region: 'eu-west-1' },
+});
+//new GithubLens(app, "my-stack-PROD", {stack: 'deploy', stage: 'PROD'});
```
Comment (akash1810) [inline packages/cdk/bin/cdk.ts:12]: Don't think we need this line?

Comment (JuliaBrigitte) [reply packages/cdk/bin/cdk.ts]: removed the line, but can not merge on my own because not 100% sure what we need to delete after merging.

**Pull request merged**